### PR TITLE
Add Readme

### PR DESCRIPTION
**Pull request by tphalp**
This adds basic information to the UCP Project page on GH.
Resolves UCP-27

### CODE REVIEW

File: profile/README.md [hunk 1]
```diff
@@ -1,12 +1,37 @@
-## Hi there 👋
+# Universal Connect Project
+
+Welcome to the code repository for the Universal Connect Project. Here you will find code, architectural design,
```
Comment (wesrisenmay-mx) [inline profile/README.md]: Welcome to the github project for

File: profile/README.md [hunk 2]
```diff
@@ -1,12 +1,37 @@
...
+their purpose.
+
+### Repositories
+
+- [UCW App](https://github.com/Universal-Connect-Project/ucw-app) - This is the main codebase that anyone wanting to 
+    implement the Universal Connect Widget will need. Please see the [UCW App README](https://github.com/Universal-Connect-Project/ucw-app/blob/main/README.md) for more information. 
+- [Widget Component](https://github.com/Universal-Connect-Project/universal-connect-widget-new) - This is the Widget React Component. This is the core of the Universal Connect Widget UI, and 
+is packaged as an NPM package. It does not need to be implemented directly, rather the NPM package is used by the 
+UCW App project.
+- Widget Client Library - More info to come.
```
Comment (wesrisenmay-mx) [inline profile/README.md]: We don't need this, because we implemented it in the widget server

**Review by wesrisenmay-mx — APPROVED** [review]

File: profile/README.md [hunk 3]
```diff
@@ -1,12 +1,32 @@
...
+
+Whether you are an established financial institution with existing data providers or a one-person startup 
+the Universal Connect Project (UCP) is the right way to connect to financial services like banks, credit unions and 
+wealth managers.
+
+More info at [https://universalconnectproject.org](https://universalconnectproject.org)
+
+## Documentation
+
+The documentation for the UCW is located at [https://docs.universalconnectproject.org](https://docs.universalconnectproject.org) (Coming soon...).
```
Comment (emilysallstrom) [inline profile/README.md:18]: Everything up until this point mentions UCP, this is the first time we mention UCW. I'm thinking we change this to UCP instead?

File: profile/README.md [hunk 4]
```diff
@@ -1,12 +1,32 @@
...
+Here you can find an implementation guide, API specifications, and more.
+
+## Project Structure
+
+The Universal Connect Project consists of a number of repositories. The following is a list of the repositories, and 
+their purpose.
+
+### Repositories
+
+- [UCW App](https://github.com/Universal-Connect-Project/ucw-app) - This is the main codebase that anyone wanting to
```
Comment (emilysallstrom) [inline profile/README.md:28]: Side note: I think this one makes sense to keep as UCW, since we spell out the full name as part of the description

**Pull request merged**